partner_industry_secondary/readme/CONTRIBUTORS.rst

```diff
@@ -7,3 +7,4 @@
 * Jordi Ballester Alomar <jordi.ballester@eficent.com>
 * Miquel Raïch <miquel.raich@eficent.com>
 * Cristina Martin R.
+* Eduardo de Miguel `Moduon <https://www.moduon.team/>`_
```

Comment:
I think [the official template suggest to put the company within parenthesis](https://github.com/OCA/maintainer-tools/blob/cc63f71d3da20629c71cd095cdec394291f22713/template/module/readme/CONTRIBUTORS.md):

```suggestion
* Eduardo de Miguel (`Moduon <https://www.moduon.team/>`__)
```